src/App.tsx

```diff
@@ -20,12 +34,13 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                onChange={(event) => setQuery(event.target.value.toLowerCase())}
```

Comment:
```suggestion
                onChange={(event) => setQuery(event.target.value)}
```
save it in the normal case, you anyway do `.toLowerCase()` in `getSortedMovies`